Documentation/README.md

```diff
@@ -1,6 +1,26 @@
 Documents Index
 ===============
 
-- [Developer Guide](developer-guide.md)
+## Overview and general information
+
 - [Intro to .NET Core CLI](intro-to-cli.md)
-- [Addressing Incremental Compilation Warnings](addressing-incremental-compilation-warnings.md)
+- [CLI UX Guidelines](cli-ux-guidelines.md)
+- [.NET Core native pre-requisities document](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md)
+- [Roadmap and OS support](https://github.com/dotnet/core/blob/master/roadmap.md)
+- [Comprehensive CLI documentation](https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/tools/)
+
+## Troubleshooting and issues reporting
+
+- [CLI Known Issues](https://github.com/dotnet/core/blob/master/cli/known-issues.md)
+- [Filing migration issues](migration-issues.md)
+
+## Specifications
+
+- [CLI installation scenarios](specs/cli-installation-scenarios.md)
+- [Corehost specification](specs/corehost.md)
+- [Runtime configuration file specification](specs/runtime-configuration-file.md)
+
+## Working with the CLI repo
```

Comment:
😄 The worst part I believe I made the change in dotnet/coreclr that introduced those folder names. 

Fixed.